src/platforms/stm32/src/main.c

```diff
@@ -170,10 +201,31 @@ int main()
 
     AVM_LOGI(TAG, "Starting: %s...\n", startup_module_name);
```

Comment:
For consistency with ESP32 and Pico, can we add:
```
fprintf(stdout, "---\n");
```

(and anything else that makes sense from those platforms)?

Reply:
Done.